assets/js/common.js

```diff
@@ -912,6 +912,33 @@ var hivepress = {
 				this.select();
 				document.execCommand('copy');
 			});
+
+			form.find('input[type="password"]').on('input', function(){
```

Comment:
I haven't checked this in deptch, but make sure that this is the minimum required JS code, looks hard-coded but maybe WordPress doesn't load this logic on the front-end.